plugins/jira-plugin/src/main/java/io/qameta/allure/jira/JiraExportUtility.java

```diff
@@ -0,0 +1,162 @@
+/*
+ *  Copyright 2019 Qameta Software OÜ
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License");
+ *  you may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ */
+package io.qameta.allure.jira;
+
+import io.qameta.allure.core.LaunchResults;
+import io.qameta.allure.entity.ExecutorInfo;
+import io.qameta.allure.entity.Link;
+import io.qameta.allure.entity.Statistic;
+import io.qameta.allure.entity.Status;
+import io.qameta.allure.entity.TestResult;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.util.Arrays;
+import java.util.Collection;
+import java.util.List;
+import java.util.Optional;
+import java.util.stream.Collectors;
+import java.util.stream.Stream;
+
+/**
+ * @author SeleniumTestAB.
+ * <p>
+ * Utility Class for Jira Export Plugin
+ */
+
+public final class JiraExportUtility {
+
+    private static final String EXECUTORS_BLOCK_NAME = "executor";
+    private static final Logger LOGGER = LoggerFactory.getLogger(JiraExportPlugin.class);
+
+    private JiraExportUtility() {
+    }
+
+
+    public static void handleFailedExport(final List<JiraExportResult> exportResults) {
```

Comment:
Why you are using it like static method? May be should move it to the used class

Reply:
Moved to the Plugin class and changed logic to return a list of failed exports for later use